50SAT/SPEX$50SAT.tex

```diff
@@ -0,0 +1,353 @@
+Once the primary electronics are ready, the software should be tested for functionality on the system hardware.
+
+The timeline below is preliminary:
+
+\begin{table}
+\centering
+ \begin{tabular}{|c | c | c|}
+ \hline
+ Phase & Task & Duration \\ [0.5ex]
+ \hline\hline
+ 1 & Review the existing \$50SAT designs and materials &  no more than 2 weeks \\
+ \hline
+ 2 & Subsystem development & 6 weeks \\
+ \hline
+ 2 & Order PCB design and/or assembly & 6 weeks \\
+ \hline
+ 2 & Review changes to mechanical strcuture and order materials & no more than 2 weeks \\
+ \hline
+ 2 & Testing of individual subsystems & 2 weeks \\
+ \hline
+ 3 & System Assembly & 1 week  \\
+ \hline
+ 4 & System testing & 2 weeks  \\
+ \hline
+ 5 & Generate documentation and delivery to SPEX & 1 week  \\
+ \hline
+\end{tabular}
+\label{tab:timeline}
+\end{table}
+
+\begin{itemize}
+  \item Review the existing \$50SAT designs and materials - no more than 2 weeks
+  \item Subsystem development - concurrent
+  \begin{enumerate}
+    \item Order PCB design and/or assembly - 6 weeks
+    \item Review changes to mechanical strcuture and order materials - no more than 2 weeks
+    \item Setup software environment and prototyping test bed - no more than 4 weeks
+  \end{enumerate}\
+  \item Testing of individual subsystems - 2 weeks
+  \item System Assembly - 1 week
+  \item System testing - 2 weeks
+  \item Generate documentation and delivery to SPEX
+\end{itemize}
+
+Schedule slip is tolerable, since there are few external considerations within the fall semester, e.g. a competition or public display event.
+\section{Externalities}
+  % Things not directly related to the work or outcomes, but related to the project as a whole.
+\subsection{Prerequisite Skills}
+  % Which skills do team members need to have before work can start (not including skills that will be learned ``on the job'')?
+Firstly, this project requires technical skillsets across multiple disciplines in order to be a success.
+The members that make up this project team will need applicable skills to their working domain and general awareness of the space environment and constraints.
+
+Electrical team members should have basic soldering skills, with at least one member familiar with PCB design and layout.
+Each board is flight-proven, but should be verified by members with applciable electrical engineering knowledge.
+Assembly of those boards will require SMT mounting skills.
+Also, prior experience working with solar cells is a plus.
+
+The frame and interal structure of the satellite requires mechanical engineering design and fabrication skills.
+Mechanical team members should be able to machine required components and finish final assembly.
+At least one member should have the ability to run structural and vibrational FEA on the system designs.
+At least one member should have the ability to run thermal FEA.
```

Comment:
This period should end with `\@` after it to be perfectly correct, but here it doesnt really matter since you start a new line. This is another style error caught by  [linter-chktex](https://atom.io/packages/linter-chktex)